<?xml version="1.0" encoding="UTF-8"?>

<launch>
    <arg name="initial_x" default="0" />
    <arg name="initial_y" default="8" />
    <arg name="des_x" default="0" />
    <arg name="des_y" default="-4" />
    <param name="initial_x" value="$(arg initial_x)" />
    <param name="initial_y" value="$(arg initial_y)" />
    <param name="des_pos_x" value="$(arg des_x)" />
    <param name="des_pos_y" value="$(arg des_y)" />
    <node pkg="bug_alg" type="follow_wall.py" name="wall_follower" />
    <node pkg="bug_alg" type="go_to_point_step.py" name="go_to_point_step" />
    <node pkg="bug_alg" type="distbug_step.py" name="distbug_step" output="screen" />
</launch>
